swim/node_bootstrap.go

```diff
@@ -18,28 +18,49 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package util
+package swim
```

Comment:
Good naming! I like how you named it specific to bootstrap rather than applying a catchall `util` name to it.